internal/hardware/validator.go

```diff
@@ -219,10 +227,20 @@ func (v *validator) getPreflightOCPRequirements(cluster *common.Cluster) (*model
 	}, nil
 }
 
-func (v *validator) getOCPRequirementsForVersion(openShiftVersion string) *models.VersionedHostRequirements {
-	requirements, err := v.VersionedRequirements.GetVersionedHostRequirements(openShiftVersion)
+func (v *validator) updateSingleNodeHwRequirements(requirements *models.VersionedHostRequirements) {
+	requirements.MasterRequirements.CPUCores = v.ValidatorCfg.MinCPUCoresSno
+	requirements.MasterRequirements.RAMMib = conversions.GibToMib(v.ValidatorCfg.MinRamGibSno)
```

Comment:
Shre this will do the trick?
Currently, it seems that the minimum requirement is the worker/default 8GiB

Reply:
yeah it will, tested it